server/game/cards/19-JS/Brimstone.js

```diff
@@ -0,0 +1,28 @@
+const DrawCard = require('../../drawcard.js');
+
+class Brimstone extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: card => card.getType() === 'character' && card.controller === this.controller && card.hasTrait('Sand Snake'),
+            effect: ability.effects.modifyStrength(1)
+        });
+        
+        this.action({
+            title: 'Return Sand Snake',
+            phase: 'challenge',
+            cost: [
+                ability.costs.kneelSelf(),
+                ability.costs.returnToHand(card => card.getType() === 'character' && card.controller === this.controller && card.hasTrait('Sand Snake'))
+            ],
+            handler: context => {
+                this.game.addGold(this.controller, 1);
```

Comment:
This and the message should be `context.player`.

Reply:
will fix this